src/components/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -447,7 +408,7 @@ export default {
 					fetchPolicy: 'network-only',
 				}).then(data => {
 					// need to update this.itemsInBasket here.
-					this.itemsInBasket = _get(data, 'data.shop.basket.items.values');
+					this.basketedLoans = _get(data, 'data.shop.basket.items.values');
```

Comment:
I think that the itemsInBasket prop will be updated by the watchQuery for the borrower profile. Did you see that being updated or was this line required for it to update?